### PR TITLE
Add Block course in the eighth section

### PR DESCRIPTION
**Pull request by Flasto4ka**
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -439,6 +439,44 @@ <h1 class="banner__title">
...
+                  <button class="course__button">записаться</button>
+                </div>
+
+                <div class="course__line"></div>
+              </div>
+            </div>
+
+            <div class="column-lg-6">
+              <div class="course__logo">
+                <img src="img/Univercity.svg" alt="Univercity" />
```
Comment (flast1k) [inline index.html:474]: university

File: index.html [hunk 2]
```diff
@@ -439,6 +439,44 @@ <h1 class="banner__title">
...
+                    <img src="img/adress.svg" alt="adress" />
+                    <div class="course__text">
+                      г. Нижний Новгород, проспект Гагарина, 23, корпус 2
+                    </div>
+                  </div>
+
+                  <button class="course__button">записаться</button>
+                </div>
+
+                <div class="course__line"></div>
```
Comment (flast1k) [inline index.html:468]: hr

File: index.html [hunk 3]
```diff
@@ -439,6 +439,44 @@ <h1 class="banner__title">
...
+
+      <!-- Block course -->
+
+      <section class="course">
+        <div class="container">
+          <div class="row">
+            <div class="column-lg-6">
+              <div class="course__square">
+                <div class="course__information">
+                  <div class="course__title">Записаться на курс</div>
```
Comment (flast1k) [inline index.html:451]: h2. The same for the other sections

File: index.html (hunk 1 above)
Comment (Flasto4ka) [reply index.html:474]: +

File: index.html (hunk 2 above)
Comment (Flasto4ka) [reply index.html:468]: +

File: index.html (hunk 3 above)
Comment (Flasto4ka) [reply index.html:451]: +

**Review by flast1k — APPROVED** [review]

**Pull request merged**